DOC_EXTERNAL_JOURNAL_REST_API.md

```diff
@@ -0,0 +1,313 @@
+# SORMAS REST API for external symptom journals
+
+This guide explains:
+ * why this API is needed
+ * how to access the API
+ * the data structures involved
+ * the endpoints and their purpose, together with explanations of their parameters, response types and examples.
+ * how registering a SORMAS person as an external journal patient could work
+ * some usage notes and how the data is used in SORMAS
+ 
+### Motivation
+SORMAS provides the possibility to register contacts for confirmed cases. For some diseases (e.g. COVID19)
+these contacts are contacted on a daily basis, to check their symptoms, temperature etc.<br/> SORMAS currently 
+has to some extent this functionality: each contact has a follow-up period, during which we can register
+visits, containing the status and symptoms of the person.<br/>
+There exist other applications built specifically for this purpose, which have additional features, 
+automatically sending symptom questionnaires to contacts etc. It is desirable to integrate these other systems
+with SORMAS, to enable better tracking and collecting more accurate data.
+The purpose of this API is to enable communication between SORMAS and other symptom journals. 
+
+### How to access the API
+Only users with the role ``REST_EXTERNAL_VISITS_USER`` are authorized to use the endpoints. If you would like to 
+receive access, please contact the team to have a user set up. <br/>
```

Comment:
Maybe we should specify how to contact the team. With team, you mean developer team, right? We could link to our gitter.